src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponentSpec.scala

```diff
@@ -298,6 +308,7 @@ class ClusterComponentSpec extends TestComponent with FlatSpecLike with CommonTe
   }
 
   it should "stop and start a cluster" in isolatedDbTest {
+    lazy val dateaccessed = Instant.now()
```

Comment:
It'd be clearer to name the cluster vals, `c1`, `c2`, `c3`, something like `initialCluster`, `stoppedCluster`, `runningCluster`, respectively.